tests/test_logics.py

```diff
@@ -128,10 +139,12 @@ def test_same_environ(self, monkeypatch, env_file):
     def test_multiple_vars_with_prefix(self, monkeypatch, env_file):
         """Dumper with prefix option should return all prefixed variables."""
         monkeypatch.setattr(
-            dumper, 'environ', multiple_variables_with_prefix(),
+            dumper,
+            'environ',
+            multiple_variables_with_prefix(),
         )
         dump_result = dumper.dump(template=env_file, prefixes=['SECRET_'])
 
         # Only prefix should be changed, other parts should not:
-        assert dump_result['DJANGO_SECRET_KEY'] == 'test'
-        assert dump_result['SECRET_VALUE'] == 'value'
+        assert dump_result['DJANGO_SECRET_KEY'] == 'test'  # noqa: S105
```

Comment:
after #385 we could use syrupy for `dump_result` and then also get rid of the `S105` directive overrides

Reply:
good idea!